### PR TITLE
add constraint for websockets

### PR DESCRIPTION
**Pull request by vcidst**
**Proposed changes**:
- Addresses https://github.com/sanic-org/sanic/issues/2733

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -2620,7 +2620,7 @@ multidict = ">=5.0,<6.0"
 sanic-routing = ">=0.7,<1.0"
 ujson = {version = ">=1.35", markers = "sys_platform != \"win32\" and implementation_name == \"cpython\""}
 uvloop = {version = ">=0.5.3", markers = "sys_platform != \"win32\" and implementation_name == \"cpython\""}
-websockets = ">=10.0"
+websockets = ">=10.0,<11.0"
```
Comment (ancalita) [inline poetry.lock:2623]: To double-check, installing from this lock file in a fresh venv with python 3.10 and then running any rasa command completes successfully?

Comment (vcidst) [reply poetry.lock:2623]: Yes, it does. I created a venv in Python 3.10 to validate it just now as well

**Review by radovanZRasa — APPROVED** [review]

**Review by ancalita — COMMENTED** [review]
@vcidst Could you please add a changelog of type `misc` so that both us and CSE track this change?

**Comment by vcidst** [comment]
@ancalita done, does that mean the CI will run again? 😞😨

**Comment by ancalita** [comment]
> @ancalita done, does that mean the CI will run again? 😞😨

Yeah 😬 we should probably open a ticket to include changelog changes in that category of docs changes that only trigger the docs jobs, (rather than all 100+ jobs).

**Comment by ancalita** [comment]
@vcidst Thinking again of this poetry lock change, I still think we should add `websockets` with the constraint in `pyproject.toml`, reason being that if in the near future the lock file gets refreshed because of some other dependency we need to update on `3.5.x`, this change will be lost.

**Comment by vcidst** [comment]
You're right, I can add `websockets` as a direct dependency with this constraint. This will be temporary though because,
- Sanic will fix this soon for `sanic==21.12.0`
- And, we'll upgrade to `sanic==22.12.0` or `sanic==23.3.0`

I'll make the changes right away

**Review by ancalita — APPROVED** [review]

**Pull request merged**